topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -221,8 +223,18 @@ public void handle(Long oldTimerId) {
                         if (!stopped) {
                             timerId = null;
                             boolean isInitialReconcile = oldTimerId == null;
+                            Timer.Sample reconciliationTimerSample = Timer.start(topicOperator.getMetrics().meterRegistry());
                             topicOperator.reconcileAllTopics(isInitialReconcile ? "initial " : "periodic ").setHandler(result -> {
+                                topicOperator.getPeriodicReconciliationsCounter().increment();
+                                if (result.failed()) {
+                                    topicOperator.incrementFailedReconciliationsCounter();
+                                    reconciliationTimerSample.stop(topicOperator.getReconciliationsTimer());
+                                } else {
+                                    topicOperator.incrementSuccessfulReconciliationsCounter();
+                                    reconciliationTimerSample.stop(topicOperator.getReconciliationsTimer());
+                                }
                                 if (isInitialReconcile) {
+                                    topicOperator.incrementSuccessfulReconciliationsCounter();
```

Comment:
Doesn't this and the line 233 increment the counter twice in a sinfle recomciliation?

Reply:
I think it does. I had a problem the count did not match. For example `startedReconciliations=5`, `periodicalReconciliations=7`, `sucesfulReconciliations=11`. I have no idea where is the one lost reconciliation.